### PR TITLE
109 contentshape for environment model

### PR DESCRIPTION
**Pull request by MircoNierenz**
# Description

add contentshape to environment model

## Type of change

Please delete options that are not relevant.

- [x] New type (non-breaking change which adds a type)
- [ ] Change (non-breaking change or fix on an existing type)
- [ ] Breaking change (Change that would cause existing Self Descriptions not to be accepted anymore by a Federated Catalogue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any
relevant details for your test configuration

- SD creation Wizard
- https://shacl-playground.zazuko.com/
- https://github.com/GAIA-X4PLC-AAD/provider-tools/tree/main/jsonLD_creator

## Checklist

- [x] My code follows the modelling guidelines of this project
- [x] I have performed a self-review of my own changes

### CODE REVIEW

**Comment by MircoNierenz** [comment]
add targetclasses to ontology file

File: environment-model/environment-model_shacl.ttl [hunk 1]
```diff
@@ -29,24 +29,31 @@ environment-model:EnvironmentModelShape a sh:NodeShape ;
...
             sh:minCount 1 ;
             sh:node environment-model:FormatShape ;
             sh:name "format object" ;
             sh:description "format object with properties for format informations" ;
-            sh:order 5 ;
+            sh:order 6 ;
             sh:path environment-model:format ],
         [ sh:maxCount 1 ;
-            sh:minCount 0 ;            
+            sh:minCount 1 ;
```
Comment (jtdemer) [inline environment-model/environment-model_shacl.ttl:52]: Bei hdmap ist es auch verpflichtend ha ich gesehen. Wie ist das vorgehen bei synthetischen Daten?

File: environment-model/environment-model_shacl.ttl [hunk 2]
```diff
@@ -70,6 +77,25 @@ environment-model:FormatShape a sh:NodeShape ;
             sh:path environment-model:version ] ;
     sh:targetClass environment-model:Format .
 
+environment-model:ContentShape a sh:NodeShape ;
```
Comment (jtdemer) [inline environment-model/environment-model_shacl.ttl:80]: Danke fürs hinzufügen der ContentShape

**Review by robertschubert — APPROVED** [review]
approved

**Review by jtdemer — APPROVED** [review]

**Pull request merged**